### PR TITLE
Fix setup

### PR DESCRIPTION
**Pull request by kgao**
Update deprecated react package, and remove cross IDE xcode setup.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,8 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
+*.xcodeproj/
+DerivedData/
```
Comment (kgao) [inline .gitignore:24]: 1. Exclude the Xcode project directory. 
2. Exclude the default directory created for object files and other build data.

File: ios/Podfile [hunk 2]
```diff
@@ -26,7 +26,7 @@ target 'covidsafe' do
   pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
   pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
   pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
-  pod 'ReactCommon/jscallinvoker', :path => "../node_modules/react-native/ReactCommon"
+  pod 'ReactCommon/callinvoker', :path => "../node_modules/react-native/ReactCommon"
```
Comment (kgao) [inline ios/Podfile:29]: RN v0.62 
jscallinvoker version is deprecated, use ReactCommon/callinvoker

**Review by kgao — COMMENTED** [review]
If we decided to use RN0.62 and up, need this PR to be able to install updated pod package correctly.

File: .gitignore (hunk 1 above)
Comment (kumpera) [reply .gitignore:24]: We have `ios/covidsafe.xcodeproj`  checked in because we have custom stuff in it.

**Comment by kumpera** [comment]
@kgao are you still interested in this PR? There's one issue to address before it can be merged.

**Review by kumpera — CHANGES_REQUESTED** [review]
We can't excluse xcodeproj as we have stuff checked in there.

**Pull request closed without merging**